### PR TITLE
Update bind-get.md

### LINKED ISSUE #14150: Add more details to security warning.
https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-3.0&tabs=visual-studio#write-a-basic-form
contains security warning ("For security reasons, you must opt in to binding GET request data...") which could be explained more detailed with example, etc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 164d7efe-01bf-0b26-7a23-f10704f338f5
* Version Independent ID: b3e2bc87-1582-2d34-c7c2-7c8e901ac771
* Content: [Introduction to Razor Pages in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-3.0&tabs=visual-studio#write-a-basic-form)
* Content Source: [aspnetcore/razor-pages/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/razor-pages/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-razorpages**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #14150

### CODE REVIEW

File: aspnetcore/includes/bind-get.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 >
 > To bind a property on `GET` requests, set the [[BindProperty]](/dotnet/api/microsoft.aspnetcore.mvc.bindpropertyattribute) attribute's `SupportsGet` property to `true`:
 > `[BindProperty(SupportsGet = true)]`
+> For more information, see [YouTube, bind on GET](https://www.youtube.com/watch?v=p7iHB9V-KVU&feature=youtu.be&t=54m27s)
```
Comment (guardrex) [inline aspnetcore/includes/bind-get.md]: Hum .... this is strange (I mean the prior lines). Recommend this (it's a cut-'n-paste for Lines 4-6 if you want it) ...

> To bind a property on \`GET\` requests, set the \[[BindProperty]](xref:Microsoft.AspNetCore.Mvc.BindPropertyAttribute) attribute's \`SupportsGet\` property to \`true\`:
>
> \`\`\`csharp
> \[BindProperty(SupportsGet = true)]
> \`\`\`
>
> For more information, see \[ASP.NET Core Community Standup: Bind on GET discussion (YouTube)](https://www.youtube.com/watch?v=p7iHB9V-KVU&feature=youtu.be&t=54m27s).

**Comment by guardrex** [comment]
:smile: it was cut-'n-paste directly from my comment. :smile:

**Comment by guardrex** [comment]
ooooops ... look at the diff. My suggestion was a cut-'n-paste directly from the GH comment :point_up:. Your cut from the comment edit (looks like that's what happened) grabbed the escapes.

**Comment by Rick-Anderson** [comment]
Arg, Do you miind fixing it?

**Comment by guardrex** [comment]
Sure thing.

**Pull request merged**